containers/src/impl/Kokkos_UnorderedMap_impl.hpp

```diff
@@ -76,19 +75,19 @@ struct UnorderedMapErase {
   using size_type       = typename map_type::size_type;
   using key_type        = typename map_type::key_type;
   using value_type      = typename map_type::impl_value_type;
+  using policy_type =
+      Kokkos::RangePolicy<execution_space, IndexType<size_type>>;
 
   map_type m_map;
 
-  UnorderedMapErase(map_type const& map) : m_map(map) {}
-
-  void apply() const {
+  void apply(const execution_space& space = execution_space{}) const {
```

Comment:
I would similarly avoid the default argument here.

Reply:
Indeed, consider it done :wink: